trafficcloud/video.py

```diff
@@ -149,3 +135,124 @@ def get_resolution(videopath):
+
+def create_video_from_image(folder, image_filename, video_filename, duration):
+    subprocess.call(["ffmpeg", 
+        "-loop", "1", 
+        "-i", os.path.join(folder, image_filename), 
+        "-c:v", "libx264",
+        "-t", str(duration),
+        "-pix_fmt", "yuv420p", 
+        os.path.join(folder, video_filename)])
+
+def combine_videos(videos_folder, temp_video_prefix, filename):
+    # The only way I could find to join videos was to convert the videos to .mpg format, and then join them.
+    # This seems to be the only way to keep ffmpeg happy.
+
+    # Using Popen seems to be necessary in order to pipe the output of one into the other
+    p1 = subprocess.Popen(['cat']+get_list_of_files(videos_folder, temp_video_prefix, "mpg"), stdout=subprocess.PIPE)
+    p2 = subprocess.Popen(['ffmpeg', 
+        '-f', 'mpeg', 
+        '-i', '-', 
+        '-qscale', '0', 
+        '-vcodec', 'mpeg4', 
+        os.path.join(videos_folder, filename)], stdin=p1.stdout, stdout=subprocess.PIPE)
+
+    p1.stdout.close()  # Allow p1 to receive a SIGPIPE if p2 exits.
+    p2.wait()
+
+### Images Helpers
+
+def renumber_frames(folder, start_frame, prefix, extension):
+    temp_folder = os.path.join(folder, "temp")
+    # Make the temp folder if it doesn't exists 
+    if not os.path.exists(temp_folder):
+        os.makedirs(temp_folder)
+
+    # Rename them all to 'new-image-x' in order to not interfere with the current 'image-x'
+    for file in os.listdir(folder):
+        if file.startswith(prefix):
+            rest = file[len(prefix):]
+            s = rest.split('.')
+            if len(s) == 2:
+                e = s[1]
+                if e == extension:
+                    number = rest.split('.')[0]
+                    try:
+                        num = int(number) - start_frame
+                    except:
+                        print("Couldn't parse to int: "+file+" from prefix: "+prefix)
+                    if num < 0:
+                        raise Error()
+                    new_file = prefix+str(num)+'.'+extension
+                    os.rename(os.path.join(folder, file), os.path.join(temp_folder, new_file))
+
+    # Rename the 'new-image-x' to 'image-x'
+    for file in os.listdir(temp_folder):
+        os.rename(os.path.join(temp_folder, file), os.path.join(folder, file))
+
+    os.removedirs(temp_folder)
+
+def convert_frames_to_video(video_path, images_folder, videos_folder, images_prefix, filename, pts_multiplier):
+    subprocess.call(["ffmpeg", 
+        "-framerate", get_framerate(video_path), 
+        "-i", os.path.join(images_folder, images_prefix+"%d.png"),
+        "-filter:v", "setpts={:0.1f}*PTS".format(pts_multiplier),
+        "-c:v", "libx264", 
+        "-pix_fmt", "yuv420p", 
+        os.path.join(videos_folder, filename)])
+
+def create_title_slide(width, height, save_path, object_id1, object_id2, fontsize=None, textcolor='#FFFFFF', facecolor='#000000'):
+    """
```

Comment:
Above might be good for when we're doing the official API docs, but most Python functions have docstrings below function header based on my experience.